.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
+env:
+- COVERAGE=
```

Comment:
@muffinresearch the goal was to run the tests with and without the code coverage instrumentation applied.

I'm not sure if we want to keep it, the main reason can be to spot any failure which doesn't happen if the code coverage instrumentation has not been applied (it should not happen, but it is not completely impossible)